[FLINK-19527][Doc] Common errors when running with docker on windows (#14002)
apache/flink: docs/try-flink/table_api.md

If you get stuck, check out the [community support resources](https://flink.apache.org/community.html).
In particular, Apache Flink's [user mailing list](https://flink.apache.org/community.html#mailing-lists) consistently ranks as one of the most active of any Apache project and a great way to get help quickly. 

> ### Docker on windows

Reviewer: We capitalize all proper words in titles along with a new line between the header and the first 

```suggestion
### Docker on Windows

```

> if you're running docker on windows and your data generator container is failing to start then please ensure that you're using right shell.

Reviewer: ```suggestion
If you're running docker on windows and your data generator container is failing to start then please ensure that you're using the right shell.
```

Grammar

> For example `docker-entrypoint.sh` for `table-walkthrough_data-generator_1` container requires bash shell. 

Reviewer: ```suggestion
For example, `docker-entrypoint.sh` for `table-walkthrough_data-generator_1` container requires bash. 
```

If you don't have it, it will throw an error `standard_init_linux.go:211: exec user process caused "no such file or directory"`. 

> So a workaround is to swith shell to `sh` from `bash` on the first row in `docker-entrypoint.sh` to make it work.

Reviewer: ```suggestion
A workaround is to switch shell to `sh` from `bash` on the first row in `docker-entrypoint.sh`.
```
